docs-guides/source/convert-llm.md

```diff
@@ -0,0 +1,151 @@
+# Converting a Large Language Model
+
+The following example shows how to convert into Core ML an [OpenELM](https://huggingface.co/apple/OpenELM) model trained using PyTorch. OpenELM is a family of Open Efficient Language Models. The small size variants are suitable for mobile and embedded language applications.
```

Comment:
This sentence is a bit awkward. I would rewrite: The following example shows how to convert a PyTorch OpenELM model to Core ML.

Reply:
Revised as suggested